FiveCalls/FiveCalls/Issue.swift

```diff
@@ -42,6 +43,6 @@ extension Issue : JSONSerializable {
         
         let contacts = contactsJSON.flatMap({ Contact(dictionary: $0) })
         
-        self.init(id: id, name: name, reason: reason, script: script, contacts: contacts)
+        self.init(id: id, name: name, reason: reason, script: script, contacts: contacts, madeCalls: false)
```

Comment:
Do you want to handle that as a separate PR so we can merge this in?  (last things I think are the padding + status bar style, which should be pretty quick)